bom/pom.xml

```diff
@@ -74,4 +74,11 @@
     </dependencies>
   </dependencyManagement>
 
+  <distributionManagement>
+    <repository>
+      <id>sonatype-nexus-staging</id>
+      <name>Nexus Release Repository</name>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+  </distributionManagement>
```

Comment:
Stripping the parent means that Maven doesn’t know where to upload releases to. Ick.